src/pages/Stylesheets/Stylesheets.js

```diff
@@ -42,11 +42,11 @@ const Stylesheets = () => {
                     <TextField error label={inputsType.ERROR} helperText="error" />
                     <TextField label={inputsType.DISABLED} disabled={true} />
 
-                    <label htmlFor="upload-photo">
+                    <label htmlFor="upload-file">
                         <input
-                            style={{ display: 'none' }}
-                            id="upload-photo"
-                            name="upload-photo"
+                            className={classes.input}
```

Comment:
why name is input: and why display: none?

Reply:
FAB (floating action button) is actual look of input, but it dont have functionality of this element so I hide input to place him behind fab.
Consequently we have good looking file input.

_Q1:  whats the right name of this class should be?_
_Q2 : is there any better way to style file input?_